htdocs/commande/card.php

```diff
@@ -60,7 +61,7 @@
 }
 
 // Load translation files required by the page
-$langs->loadLangs(array('orders', 'sendings', 'companies', 'bills', 'propal', 'deliveries', 'products', 'other'));
+$langs->loadLangs(array('orders', 'sendings', 'companies', 'bills', 'propal', 'deliveries', 'products', 'other', 'errors'));
```

Comment:
The goal of the errors lang file is to contains error message that are loaded only when required.
Can you move the load of lang file
$langs->load("errors");
just before the code that need the error keys, so before the line 
$value = $langs->trans('ErrorNoValueForRadioType');

Reply:
ok done